tests/python/contrib/test_ethosn/test_networks.py

```diff
@@ -104,17 +96,16 @@ def test_mobilenet_v1():
     # codegen, which could come about from either a change in Support Library
     # version or a change in the Ethos-N codegen. To update this requires running
     # on hardware that isn't available in CI.
-    hw = ethosn_available()
     _test_image_network(
         model_url="https://storage.googleapis.com/download.tensorflow.org/"
         "models/mobilenet_v1_2018_08_02/mobilenet_v1_1.0_224_quant.tgz",
         model_sub_path="mobilenet_v1_1.0_224_quant.tflite",
         input_dict={"input": (1, 224, 224, 3)},
         compile_hash="81637c89339201a07dc96e3b5dbf836a",
         output_count=1,
-        run=(hw == Available.SW_AND_HW),
         host_ops=3,
         npu_partitions=1,
+        run=True,
```

Comment:
I thought we are not running anything ?

Reply:
We do run that particular one because it's small :)

Reply:
I think its worth mentioning the functionality of inference_result as registered as ""relay.ethos-n.test.infra.inference_result" has two faces :) -- based on the availability of the hardware, in the tei. 
Otherwise, its hard to understand what's this "run" really means and the need for the assert_lib_hash.

Reply:
Updated docs to explain what run does in this context.

Reply:
Thanks! all good now.